papyrus/tests/test_init.py

```diff
@@ -8,9 +8,9 @@ def test_it(self):
         c = Configurator(autocommit=True)
         c.include(pyramid_handlers.includeme)
         c.include(papyrus.includeme)
-        self.failUnless(c.add_handler.im_func.__docobj__ is
+        self.failUnless(c.add_handler.__func__.__docobj__ is
                         pyramid_handlers.add_handler)
-        self.failUnless(c.add_papyrus_handler.im_func.__docobj__ is
+        self.failUnless(c.add_papyrus_handler.__func__.__docobj__ is
```

Comment:
In python 3, the `im_func` was renamed to `__func__` (see http://python3porting.com/stdlib.html). The `__func__` attribute is compatible is compatible with both.